adminLogin.php

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Login Form</title>
+</head>
+
+<body>
+    <?php
+    session_start();
+    if ($_SESSION['userLoggedIn']){
+        header("Location: cmsHomePage.php");
```

Comment:
have u checked this works? think i commented on it last time. im 95% sure it doesnt.